c++/src/ColumnWriter.cc

```diff
@@ -922,8 +931,8 @@ namespace orc {
       }
     };
 
-    std::map<DictEntry, size_t, LessThan> dict_;
-    std::vector<std::vector<char>> data_;
+    mutable std::vector<DictEntryWithIndex> flat_dict;
```

Comment:
Please follow the naming style.
[The format check didn't fail as expected.](https://github.com/apache/orc/actions/runs/10503087327/job/29095780106?pr=2010#step:4:66)

Reply:
I'll fix it.